preview/index.js

```diff
@@ -57,11 +56,7 @@ app.get("/pull/:pr(\\d+)", (req, res) => {
 
   fetch(`${githubApiUrl}/pulls/${pr}`)
     .then((response) => response.json())
-    .then(({ head: { ref, sha } }) => {
```

Comment:
I took this part out because, if it wouldn't be used anyway, I didn't want another security case to be opened down the line. I don't think that would happen, but better to not leave cruft anyway.